ロケール設定による警告（旧：Visual Studioでビルド時の警告

OSDN より移行。 [#35729](https://osdn.net/projects/hengband/ticket/35729)

> システムロケールが「日本」以外の場合にwarning C4566が大量に出ます。
> Windowsのシステムロケールを「日本」に設定した場合はこの警告は出ません。
> Visual Studio 2015で確認していますが、他のバージョンでも同様の警告が出るかと思います。

> この件、この前ソースを全てBOMつきUTF-8に変えてどうなったか再確認しておきたい。
> この件まだ多分直ってない、はず。

--- a/Hengband/Hengband/Hengband.vcxproj
+++ b/Hengband/Hengband/Hengband.vcxproj
@@ -117,6 +117,7 @@
       <ExceptionHandling>SyncCThrow</ExceptionHandling>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <ForcedIncludeFiles>stdafx.h</ForcedIncludeFiles>
+      <AdditionalOptions>/execution-charset:shift-jis %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
     <Link>
       <GenerateDebugInformation>true</GenerateDebugInformation>
@@ -174,6 +175,7 @@
       <LanguageStandard>stdcpplatest</LanguageStandard>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <ForcedIncludeFiles>stdafx.h</ForcedIncludeFiles>
+      <AdditionalOptions>/execution-charset:shift-jis %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
     <Link>
       <TargetMachine>MachineX86</TargetMachine>

